src/main/java/Stations.java

```diff
@@ -1,22 +1,37 @@
+import com.google.gson.annotations.SerializedName;
+import java.util.HashMap;
 import java.util.List;
 
 public class Stations {
-    List<Feature> features;
+    @SerializedName("features")
+    List<Station> stations;
 
-    public static class Feature {
+    public static class Station {
         Properties properties;
         Geometry geometry;
     }
 
     public static class Properties {
         String name;
+        String objectid;
```

Comment:
`int`

Reply:
We have mentioned now a few times in class how objectids are much better as ints